src/table/components/withSelections.jsx

```diff
@@ -11,7 +11,9 @@ export default function withSelections(CellComponent) {
     const selectionDispatch = useContextSelector(TableContext, (value) => value.selectionDispatch);
 
     const handleMouseUp = (evt) =>
-      cell.isSelectable && evt.button === 0 && selectionDispatch({ type: 'select', payload: { cell, evt, announce } });
+      cell.isSelectable &&
+      evt.button === 0 &&
+      selectionDispatch({ type: SelectionActions.SELECT, payload: { cell, evt, announce } });
```

Comment:
Have you heard of the 'action-creator' concept? it abstracts creating actions with the right type and params, it cn also help with default values and such. Define them in the same file as your actions. You can see some examples [here](https://github.com/qlik-trial/direct-query-modelling-ui/blob/main/src/state/shared.ts#L65-L87). Something like:

```typescript
function select(cell, evt, announce) {
  return 
    { type: SelectionActions.SELECT, payload: { cell, evt, announce } }
}